src/annotator/util/test/listener-collection-test.js

```diff
@@ -0,0 +1,30 @@
+import { listenerCollection } from '../../util/listener-collection';
+
+describe('register/unregister event listener', () => {
```

Comment:
As a general rule, the top-level `describe` block in a file should either be named after the class/function being tested, if there is only one and it is a reasonably unique name, or the path relative to `src/` otherwise (`annotator/util/listener-collection`) in this context. The reason is just to make it easier to know what file to go looking at if the tests fail.

Reply:
👍🏼